src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.cs

```diff
@@ -84,16 +84,20 @@ public X11XamlRootHost(X11WindowWrapper wrapper, Window winUIWindow, XamlRoot xa
 		};
 		_renderTimer.Start();
 
-		_resizeCallback = size =>
-		{
-			// copy the root window dimensions to the top window
-			using var _1 = X11Helper.XLock(TopX11Window.Display);
-			_ = XLib.XResizeWindow(TopX11Window.Display, TopX11Window.Window, (int)size.Width, (int)size.Height);
-			resizeCallback(size);
-		};
 		_closingCallback = closingCallback;
 		_focusCallback = focusCallback;
 		_visibilityCallback = visibilityCallback;
+		_configureCallback = configureCallback;
+
+		_configureTimer = new DispatcherTimer();
+		_configureTimer.Interval = new TimeSpan(1000 / 16);
```

Comment:
what does this constant represent (1000 / 16)? 

Hz?

Reply:
Not sure cc @ramezgerges

Reply:
This checks and updates window sizing /positioning details 60 times a second.